src/test/ui/issue-50781.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0038]: the trait `X` cannot be made into an object
+  --> $DIR/issue-50781.rs:21:6
+   |
+LL | impl Trait for dyn X {}
+   |      ^^^^^ the trait `X` cannot be made into an object
```

Comment:
Huh, the span here is a bit odd, no? I feel like it should underline the `dyn X`.

Reply:
(Presumably independent from this PR though)